### DISCUSSION TITLE
showing more realistic impact of variants

### DISCUSSION THREAD

**Discussion by Basu-droid** [hodcroftlab/covariants#244] [Ideas]
I think it would be quite interesting for many visitors the possibility of switching to a more realistic estimate of the impact of the variants in each country. If you have easy access to case counts, at each date you could simply multiply case count by the variant % you currently show. 
Anyway, great work! and thank you very much.

**Comment by emmahodcroft** [comment]
Hi @Basu-droid ! Thanks for the suggestion! I recognise this would be useful, though it's something we need to be careful about as we'd only want to do it for countries where we feel sequencing representative. However, I've added it to our to-do list a few days ago as an issue, and we can see if we can tackle it!
You can see it here: https://github.com/hodcroftlab/covariants/issues/243
